docs/developer/architecture/core/logging-service-configuration.asciidoc

```diff
@@ -0,0 +1,73 @@
+[[logging-service-configuration-example]]
+== Logging Service Configuration Example
```

Comment:
I think this page may make more sense under the `docs/setup` directory where we have dedicated pages for different configuration domains.

Reply:
That's a great idea! With the `logging-service-configuration` in the `docs/setup` directory, we can link to the basic examples from the main settings page. 😄 

We still can't link to the more advanced documentation yet but that will be done in a follow up PR. I've added a comment to the `logging.root` description to get that done next.